config/release_notes.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+Release::Notes.configure do |config|
```

Comment:
We should only include what we're overriding. Can comment out the default configs.